### DISCUSSION TITLE
General stammer chat

### DISCUSSION THREAD

**Discussion by ArdenButterfield** [ArdenButterfield/stammer#91] [General]
Hi folks! If you have any questions that aren't quite issues... or just want to chat... here's the place!

**Comment by Mattandball** [comment]
Hey lad! What the hell is   File "/Users/justinguo/Downloads/stammer-main/stammer.py", line 5, in <module>
    import numpy as np
ModuleNotFoundError: No module named 'numpy' I installed numpy and updated it so many times
PLS FIX

**Reply by Firepal** [reply]
Read the [main page](https://github.com/ArdenButterfield/stammer) and it will inform you that you have to use pip to install numpy yourself
